### PR TITLE
feat(pageintro): kSE3-492 Add styles to page intro

### PR DESCRIPTION
**Pull request by TreyJV**
Add the styles to the page intro section

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.scss [hunk 1]
```diff
@@ -1,10 +1,14 @@
 .page-intro {
-  height: 109px;
+  height: var(--wh-2XL);
   padding: 10px;
```
Comment (mariokleen) [inline libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.scss:3]: pm-s ?

File: libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx [hunk 2]
```diff
@@ -1,23 +1,33 @@
 import React from 'react';
-import MuiButton from '@material-ui/core/Button';
-import Grid from '@material-ui/core/Grid';
-import Paper from '@material-ui/core/Paper';
-import Typography from '@material-ui/core/Typography';
+import { styled } from '@material-ui/core/styles';
+import './PageIntroSection.scss';
+
```
Comment (mariokleen) [inline libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx:4]: maybe it's not necessary to order the imports (maybe), but I think we don't need the extra space

File: libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
-import { styled } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
 
 const Button = styled(MuiButton)({
   background: 'var(--secondary-color)',
   "&:hover": {
     background: 'var(--secondary-color-variant)',
-  }
+  },
+  marginRight: '16px',
```
Comment (mariokleen) [inline libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx:20]: shoudln't margin (structural) should be in the scss?

File: libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx [hunk 4]
```diff
@@ -158,14 +172,14 @@ export const PageIntroSection = (props: PageIntroSectionProps) => {
 
   return (
     <>
-      <Paper className="page-intro">
-        <Grid container spacing={1} justify="space-evenly">
+      <Container maxWidth="xl" className="page-intro">
```
Comment (mariokleen) [inline libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx:175]: did you try this with a range of buttons? only one and up to 3, at least?

Comment (TreyJV) [reply libs/atomic-elements/src/lib/components/PageIntroSection/PageIntroSection.tsx:175]: In this case, at this moment, we will not support the Avatar section or more than one button, so it is not necessary testing with more buttons.  🙂

**Pull request merged**